xtekky/gpt4free
# Docker Support for gpt4free

## Overview:
This PR introduces Docker support to simplify the setup and usage of the gpt4free project, making it more accessible to developers and ensuring a consistent environment.

## Changes:
- Added `Dockerfile` to define the container environment.
- Added `docker-compose.yml` for easy setup and orchestration.
- Updated `README.md` with instructions on how to use Docker to set up and run the project.

## How to Test:
1. Ensure Docker and docker-compose are installed.
2. Clone this branch.
3. Run `docker-compose up` from the project root.
4. The application should be accessible at `http://localhost:1337`.

Please review and provide feedback. If there are any issues or improvements needed, I'm open to making the necessary changes. Thank you for considering this contribution!
